### PR TITLE
Created the cardview and recycler view. This should be modified later…

### PR DESCRIPTION
**Pull request by ChristianMorup**

### CODE REVIEW

File: app/src/main/java/com/example/greenfingers/MainActivity.java [hunk 1]
```diff
@@ -2,12 +2,23 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
 
 public class MainActivity extends AppCompatActivity {
+    private RecyclerView rcvMyPlants;
+    private RecyclerView.Adapter adapter;
```
Comment (Cpilgaard) [inline app/src/main/java/com/example/greenfingers/MainActivity.java]: Please change this name to plantAdapter.

File: app/src/main/java/com/example/greenfingers/PlantAdapter.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        CardView cardView;
+
+        ViewHolder(@NonNull View itemView) {
+            super(itemView);
+            txtPlantName = itemView.findViewById(R.id.txtPlantName);
+            imgPlant = itemView.findViewById(R.id.imgPlant);
+            cardView = itemView.findViewById(R.id.plantCardView);
+        }
+    }
+
```
Comment (Cpilgaard) [inline app/src/main/java/com/example/greenfingers/PlantAdapter.java]: Please remove spacing

File: app/src/main/java/com/example/greenfingers/PlantAdapter.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            viewHolder.cardView.setElevation(i);
+        }
+    }
+
+    @Override
+    public int getItemCount() {
+        return 10;
+    }
+
```
Comment (Cpilgaard) [inline app/src/main/java/com/example/greenfingers/PlantAdapter.java]: Please remove spacing

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -7,9 +7,16 @@
...
 
     <android.support.v7.widget.RecyclerView
-        android:layout_width="395dp"
-        android:layout_height="340dp"
-        tools:layout_editor_absoluteX="8dp"
-        tools:layout_editor_absoluteY="88dp" />
+        android:id="@+id/rcvPlants"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_marginStart="8dp"
```
Comment (MikkelHarbo) [inline app/src/main/res/layout/activity_main.xml]: Not quite the best idea to "hardcode" the size of the view.  Maybe use guidelines for futher adjustments.

**Pull request merged**